src/openassetio-core/src/utils/ostream.cpp

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 The Foundry Visionmongers Ltd
+#include <openassetio/Context.hpp>
+#include <openassetio/EntityReference.hpp>
+#include <openassetio/hostApi/Manager.hpp>
+#include <openassetio/managerApi/HostSession.hpp>
+#include <openassetio/utils/ostream.hpp>
+
+#include "../src/utils/formatter.hpp"
```

Comment:
[consider] This confused me.  I think just `#include "formatter.hpp"` is better.

Reply:
Huh, i'd gotten it into my head that these _had_ to be relative like this. :shrug:

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1329/commits/667ffcd50fb62084709323736d8fbd9b236be8d8